packages/react/package.json

```diff
@@ -62,7 +62,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@slashid/slashid": "3.15.0",
+    "@slashid/slashid": "^3.16.0",
```

Comment:
Should we go with a fixed version? Just to prevent surprises from happening 😅

Reply:
Done!